service/gcs/bridge/bridge_test.go

```diff
@@ -613,10 +613,29 @@ var _ = Describe("Bridge", func() {
 				})
 				AssertNoResponseErrors()
 				AssertActivityIDCorrect()
-				// TODO: Add tests on callArgs when resizing the console is
-				// implemented.
 				It("should receive the correct values", func() {
-					//e.g. Expect(callArgs.Pid).To(Equal(101))
+					Expect(callArgs.Pid).To(Equal(101))
+					Expect(callArgs.Height).To(Equal(uint16(30)))
+					Expect(callArgs.Width).To(Equal(uint16(72)))
+				})
+			})
+			Context("the message is normal ASCII no containerid", func() {
+				BeforeEach(func() {
+					message = prot.ContainerResizeConsole{
+						MessageBase: &prot.MessageBase{
+							ActivityID: activityID,
+						},
+						ProcessID: 102,
+						Height:    80,
+						Width:     80,
+					}
+				})
+				AssertNoResponseErrors()
+				AssertActivityIDCorrect()
+				It("should receive the correct values", func() {
+					Expect(callArgs.Pid).To(Equal(102))
+					Expect(callArgs.Height).To(Equal(uint16(80)))
+					Expect(callArgs.Width).To(Equal(uint16(80)))
```

Comment:
Oops, forgot to mention this on my last review: If we're getting rid of the `id==""` check in this PR, we should probably get rid of this test.